src/plugins/layout/states/ignore.ts

```diff
@@ -30,8 +30,7 @@ export function shouldIgnoreNode(node: Node, parentInfo: INodeInfo): boolean {
             // If we capture CSSRules on style elements, ignore its text children nodes
             // Since iterating over css rules can be 100X slower on certain browsers,
             // we limit ignoring text nodes to STYLE elements with empty text content
-            let parent = node.parentNode as Element;
-            if (parent && parent.tagName === "STYLE" && parent.textContent.length === 0) {
+            if (parentInfo && parentInfo.captureCssRules) {
                 ignore = true;
             }
```

Comment:
Missing break at end of case.

Reply:
Thanks

---
In reply to: [261422832](https://github.com/Microsoft/clarity-js/pull/138#discussion_r261422832) [](ancestors = 261422832)